.github/workflows/reusable-revised-its.yml

```diff
@@ -57,6 +57,19 @@ on:
       AWS_SECRET_ACCESS_KEY:
         required: false
         type: string
+      BACKWARD_INCOMPATIBILITY_IT_ENABLED:
+        required: false
+        type: string
+        default: false
+      DRUID_PREVIOUS_VERSION:
+        required: false
+        type: string
+      DRUID_PREVIOUS_VERSION_DOWNLOAD_URL:
+        required: false
+        type: string
+      DRUID_PREVIOUS_IT_IMAGE_NAME:
```

Comment:
I think we can remove this image name. We can easily derive it from the `DRUID_PREVIOUS_VERSION`

Reply:
Done, added a job to derive it internally.